modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -86,8 +85,7 @@ const getFieldDetails = (props) => {
 
 	if (hasError) {
 		fieldDetails += Liferay.Util.escape(errorMessage);
-	}
-	else if (required) {
+	} else if (required) {
```

Comment:
@joaovictoralvesmartins, please perform source format changes in dedicated commits, like this [one](https://github.com/liferay/liferay-portal/commit/0d09f8f5330a84fb1075a25356863362a0e89b83)

Thanks

Reply:
@natocesarrego, so whenever i run `gw formatSource`, should i make a commit of it?

Reply:
Yes!

Reply:
If the generated changes modify the code not changed by you then you should create a commit only with SF, because it improves the pull request readability and comprehension to the reviewers. Actually, this is a common pattern on the pull requests of liferay-portal. :wink: